client/dashboard/sites/backups/backup-now-button.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { Button, Tooltip } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useCallback, useEffect, useState } from 'react';
+import { siteBackupsQuery } from '../../app/queries/site-backups';
+import { enqueueSiteBackup } from '../../data/site-backup';
+import { type BackupEntry } from '../../data/site-backups';
+import type { Site } from '../../data/types';
+
+interface BackupNowButtonProps {
+	site: Site;
+}
+
+type BackupState = 'default' | 'enqueued' | 'in_progress';
+
+export function BackupNowButton( { site }: BackupNowButtonProps ) {
+	const [ isEnqueued, setIsEnqueued ] = useState( false );
+	const queryClient = useQueryClient();
+
+	// Determine current state
+	const getBackupState = useCallback(
+		( backups: BackupEntry[], enqueued: boolean ): BackupState => {
+			const currentBackup = backups[ 0 ];
+			if ( currentBackup?.status === 'started' ) {
+				return 'in_progress';
+			}
+			if ( currentBackup?.status === 'queued' || enqueued ) {
+				return 'enqueued';
+			}
+			return 'default';
+		},
+		[]
+	);
+
+	// Check for in-progress backup using rewind/backups endpoint
+	const { data: rewindBackups = [] } = useQuery( {
+		...siteBackupsQuery( site.ID ),
+		refetchInterval: ( query ) => {
+			const backups = query.state.data || [];
+			const backupState = getBackupState( backups, isEnqueued );
+			// Poll when backup is enqueued or in progress
+			return backupState !== 'default' ? 2000 : false;
+		},
+	} );
+
+	const { mutate: triggerBackup, isPending } = useMutation( {
+		mutationFn: () => {
+			setIsEnqueued( true );
+			return enqueueSiteBackup( site.ID );
+		},
+		onSuccess: () => {
+			// Refresh rewind backups to check for new backup status
+			queryClient.invalidateQueries( {
+				queryKey: [ 'site', site.ID, 'rewind', 'backups' ],
+			} );
```

Comment:
We can reuse the query key like this


```suggestion
			queryClient.invalidateQueries( siteBackupsQuery( site.ID ) );
```

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/105217 -> https://github.com/Automattic/wp-calypso/pull/105217/commits/78374372f75abca90007ff80a222f0ca2397e677